### PR TITLE
Add Ruby 3.0 to CI

### PR DESCRIPTION
**Pull request by ybiquitous**
This PR just adds Ruby 3.0 to the CI matrix.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu, macos]
-        ruby: [2.5, 2.6, 2.7, head, debug, truffleruby, truffleruby-head]
+        ruby: [2.5, 2.6, 2.7, '3.0', head, debug, truffleruby, truffleruby-head]
```
Comment (ybiquitous) [inline .github/workflows/main.yml:23]: > we have to use quotes for '3.0'

-- https://github.com/ruby/setup-ruby/blob/b0055144/README.md#matrix-of-ruby-versions

**Comment by petergoldstein** [comment]
@ybiquitous Do you want to update this PR to add 3.1 as well, now that it's out?

File: .github/workflows/main.yml (hunk 1 above)
Comment (BobbyMcWho) [inline .github/workflows/main.yml:23]:
```suggestion
        ruby: [2.5, 2.6, 2.7, '3.0', 3.1, head, debug, truffleruby, truffleruby-head]
```

**Review by BobbyMcWho — APPROVED** [review]

**Comment by ybiquitous** [comment]
Sorry for the late reply.
Thanks for the review and merge!

**Pull request merged**